src/mlagility/api/script_api.py

```diff
@@ -146,8 +148,27 @@ def benchmark_script(
     models_found: Dict[str, ModelInfo] = {}
 
     for script in input_scripts:
+        script_path, targets, encoded_input = decode_input_script(script)
+
+        # Filter scripts by label
+        if labels != []:
```

Comment:
This is a spicy test because the `labels` arg defaults to `None` in this function. Should it be `if labels is not None and labels != []`?

Or if you want to play fast and loose I believe you can just do `if labels`, since that will filter out both `None` and `[]`.

Reply:
Accepted suggestion